### ISSUE TITLE
[BUG] No login to the server possible

### ISSUE THREAD

**Issue by HHZZUUJH** [astubenbord/paperless-mobile#229]
Hello,

YunoHost version: 11.1.21.4 (stable)
Paperless-ngx for YunoHost version: 1.16.2~ynh2
user to login: superuser
 
indicated error:
The request returnedan invalid status code of 302

Thanks. Best regards

**Comment by astubenbord** [comment]
Hi, sorry for the late reply. What URL are you trying to log in to paperless with? The url should NOT include the /api/. Otherwise, it may be an issue with the app currently not redirecting.

**Comment by HHZZUUJH** [comment]
Hello, thank you very much for the answer. I use the URL without /api. It is also sud my view a problem with the redirection.

**Comment by astubenbord** [comment]
Right, I had to manually enable redirects in the http client. I'll publish an update today

**Comment by astubenbord** [comment]
Hey, sorry, I've been pretty busy lately, might take a few more days until I get to releasing the update :/

**Comment by HHZZUUJH** [comment]
Hey, no problem. Now already many thanks for it

**Comment by astubenbord** [comment]
Could you check if this is fixed in the latest version (3.0.1, available here on GitHub or Google Play open Beta)?

**Comment by HHZZUUJH** [comment]
Thank you for the new information. I don't get any error message anymore with the mentioned version. The authentication process does not complete for me. But that is because of my system configuration. I use paperless-ngx (server) on a Yunohost. There I have to log on to the Yunohost domain first, and then I am redirected via SSO to the domain of the web server paperless-ngx. This works on my smartphone in the browser. Is it possible to realize this two-step login through paperless-mobil?

**Comment by astubenbord** [comment]
I've tried integrating solutions like authelia before and didn't get it to work unfortunately.

**Comment by jedie** [comment]
Maybe i have the same problem. I installed Paperless-ngx 2.1.3 and used the app version 3.1.8.
The last log in the app is:
```
2023-12-26 11:29:36.960	DEBUG   --- [      AuthenticationCubit] - login                    : Trying to log in jens@pl***de...
2023-12-26 11:29:36.961	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Adding new user jens@pl***de..
2023-12-26 11:29:36.961	DEBUG   --- [      AuthenticationCubit] - _addUser                 : Fetching bearer token from the server...
```

More info in: https://github.com/astubenbord/paperless-mobile/issues/386

**Comment by jon-alamo** [comment]
Same here. I use paperless-ngx 2.3.3 behind a SSO (authentik) and app version 3.2.0. 

```
2024-01-19 16:17:25.908 DEBUG   --- [      AuthenticationCubit] - login                    : Trying to log in joni@pa***in...
2024-01-19 16:17:25.909 DEBUG   --- [      AuthenticationCubit] - _addUser                 : Adding new user joni@pa***in..
2024-01-19 16:17:25.909 DEBUG   --- [      AuthenticationCubit] - _addUser                 : Fetching bearer token from the server...
```

**Comment by jedie** [comment]
Any news here?

**Comment by vednolacni** [comment]
I have the same problem is there any update?

**Comment by ados8** [comment]
Issues described are expected if using the native Android mobile app with Authentik in front.
I'm assuming this based on the above wording of 'behind SSO' which will commonly cause issues.
If proxy is directing traffic to Authentik for authentication before then redirecting to the site it will break the majority of native apps.
These apps typically use an API and cannot access your browsers session cookies, therefore the app can't authorise with Authentik to even reach the Paperless site.
This is why the app is stuck fetching data, it's timing out as Authentik waits for authentication.
Your options are:
- Expose Paperless directly to web without Authentik in front and use OIDC for logins
- Configure Authentik from requiring authentication on the API URL
- Access Paperless over a VPN

**Comment by alexgorbatchev** [comment]
One approach that works is to setup OIDC via Paperless config, not as a proxy. In that case the mobile app has API access. I've created a second user with a password and can signin either via Authentik when using a browser or using the second user on mobile.

**Comment by astubenbord** [comment]
Closing in favor of #374
